stash_engine/app/views/stash_engine/user_mailer/peer_review.html.erb

```diff
@@ -0,0 +1,9 @@
+<p>Dear @user.name,</p>
```

Comment:
You probably ant this to be <%= @user.name %> here, right?

Reply:
yes. discovered that and some other issues with the email/view rendering this morning.